locales-id-ID.xml

```diff
@@ -69,13 +69,13 @@
     <term name="page-range-delimiter">–</term>
 
     <!-- ORDINALS -->
-    <term name="ordinal"></term>
-    <term name="ordinal-01"></term>
-    <term name="ordinal-02"></term>
-    <term name="ordinal-03"></term>
-    <term name="ordinal-11"></term>
-    <term name="ordinal-12"></term>
-    <term name="ordinal-13"></term>
+    <term name="ordinal">bilangan bertingkat</term>
```

Comment:
It is: https://en.wikipedia.org/wiki/Ordinal_indicator#Representation_as_prefix

If we decide we want to support this, perhaps the best option (no pun intended) would be to add a toggle on `<style-options/>`. The only alternatives I can think of, having separate terms (`<term name="ordinal-suffix-01">`) or a term attribute (`<term name="ordinal-01" position="before">`) seem a lot messier.